File: testdata/PullUpMethod/Java/parent/ChildClass.java
// File: ClassExample.java
public class ChildClass extends ParentClass {
    public void childMethod() {
        System.out.println("Child method");
    }
    
    public void commonMethod() {
        System.out.println("Common method in Child");
    }
}

class ParentClass {
    // No commonMethod in ParentClass
}

File: testdata/PullUpMethod/Cpp/parent/ChildClass.cpp
// File: ClassExample.cpp
#include <iostream>

class ParentClass {
    // No commonMethod in ParentClass
};

class ChildClass : public ParentClass {
public:
    void childMethod() {
        std::cout << "Child method" << std::endl;
    }

    void commonMethod() {
        std::cout << "Common method in Child" << std::endl;
    }
};
